backend-rust/src/indexer.rs

```diff
@@ -3223,7 +3219,7 @@ async fn process_cis2_token_event(
                 token_address,
                 contract_index,
                 contract_sub_index,
-                metadata_url.as_str(),
+                metadata_url.url(),
```

Comment:
pub fn url(&self) -> &str { &self.url }

This is still just returning the string?